62.CAD/vertex_shader.hlsl

```diff
@@ -417,22 +429,68 @@ PSInput main(uint vertexID : SV_VertexID)
         }
 
         outV.position.xy = (outV.position.xy / globals.resolution) * 2.0 - 1.0;
+    }
+    else if (objType == ObjectType::CURVE_BOX)
+    {
+        outV.setColor(lineStyle.color);
+        outV.setLineThickness(screenSpaceLineWidth / 2.0f);
 
+        CurveBox curveBox;
+        curveBox.aabbMin = vk::RawBufferLoad<double2>(drawObj.geometryAddress, 8u);
+        curveBox.aabbMax = vk::RawBufferLoad<double2>(drawObj.geometryAddress + sizeof(double2), 8u);
+        for (uint32_t i = 0; i < 3; i ++)
+        {
+            curveBox.curveMin[i] = vk::RawBufferLoad<double2>(drawObj.geometryAddress + sizeof(double2) * (2 + i), 8u);
+            curveBox.curveMax[i] = vk::RawBufferLoad<double2>(drawObj.geometryAddress + sizeof(double2) * (2 + 3 + i), 8u);
```

Comment:
since this is uv space, make these curveMin/Max `uint32_t`'s and map/unmap it to/from a float in [0,1] to have ALL the precision inside the slab